src/components/Navbar/AccountAndLogo.tsx

```diff
@@ -58,7 +60,8 @@ export default function () {
       onClick={async () => await WalletStore.connect(true)}
     >
       <AccentText color="text-primary-semi-dimmed">
-        No wallet connected
+        <span className={displayFrom('body')}>No wallet connected</span>
+        <span className={displayTo('body')}>No wallet</span>
```

Comment:
Why is it `No wallet` for just this one, I'm guessing that it's too small right?

Reply:
Yes, `No wallet connected` is too long, the text jumps too much, so I decided to cut the `connected` word like this. Doing:
```TS
No wallet <span className={displayFrom('body')}>connected</span>
```
Will break the layout, the words will go like this:
```TS
No wallet
connected
```
To fix it, you need additional wrapping layers, so I decided to make two different texts